### PR TITLE
fix(CFP): pass correct parameters to submit cfp function

### PR DESCRIPTION
**Pull request by harshtandiya**
- incorrect parameters were passed to the create_submission function. Fixed that

### CODE REVIEW

**Comment by RiyaMathew-11** [comment]
Can you please mention, what were the incorrect parameters passed ?

**Review by mngshm — DISMISSED** [review]

**Comment by rahulporuri** [comment]
@Mangeshrex did you test these changes locally to ensure that the code works as expected now?

File: fossunited/fossunited/doctype/foss_event_cfp/foss_event_cfp.py [hunk 1]
```diff
@@ -205,8 +205,8 @@ def create_cfp_submission(fields):
         "doctype": "FOSS Event CFP Submission",
         "submitted_by": frappe.session.user,
     }
-    frappe.parse_json(fields).update(fields_dict)
-    doc = frappe.get_doc(fields_dict)
+    fields=frappe.parse_json(fields).update(fields_dict)
```
Comment (rahulporuri) [inline fossunited/fossunited/doctype/foss_event_cfp/foss_event_cfp.py]:
```suggestion
    fields = frappe.parse_json(fields).update(fields_dict)
```

**Comment by RiyaMathew-11** [comment]
I've replicated the error - here's a gif of that: 

FOSS-CFP-CAPTURE

With the fix in PR, the CFP got submitted: 

Screenshot 2024-07-05 at 9 19 23 PM


However the `Edit CFP` and `All Proposals` link seems broken now: 

FOSS-CFP-EDIT

This needs to be tested - in case of a false warning, seems like a route issue for me.

**Review by rahulporuri — APPROVED** [review]
Approving as @RiyaMathew-11 has tested and validated that this PR fixes the issue of not being able to submit a proposal.

**Review by RiyaMathew-11 — APPROVED** [review]
Fixes the issue of not being able to submit a proposal

**Pull request merged**